action.yml

```
@@ -1,29 +1,31 @@
 name: 'Interface exporter action'
-description: 'An action to generate npm packages based on Solidity contract interfaces'
+description: 'An action to generate npm packages based on solidity interfaces and contracts'
```

Uppercase Solidity, for example: https://github.com/defi-wonderland/solhint-plugin#overview
```suggestion
description: 'An action to generate npm packages based on Solidity interfaces and contracts'
```